OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -10,20 +10,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Fǝtḥata śǝrāy, Amharic theological treatise, list of commemorations of saints, hymns</title>
-                
-                <!-- 
-                <title xml:lang="gez" xml:id="title1">ፍትሐተ፡ ሥራይ፡, ነገረ፡ ስሕተቶሙ፡ ለመናፍቃን፡, </title>
+                <title xml:lang="gez" xml:id="title1">ፍትሐተ፡ ሥራይ፡, ነገረ፡ ስሕተቶሙ፡ ለመናፍቃን፡</title>
                 <title xml:lang="gez" corresp="#title1" type="normalized">Fǝtḥata śǝrāy, Nagara sǝḥtatomu la-manāfǝqān</title>
-                <title xml:lang="en" corresp="#title1">Dissolution of the Spell, Amharic theological treatise, Discourse on the Errors of the Heretics, List of commemorations of saints, Malkǝʾ-hymns</title>
-                -->
+                <title xml:lang="en" corresp="#title1">Dissolution of the Spell, Amharic theological treatise, Discourse on the Errors of the Heretics, 
```

Comment:
Here my feeling would be that Fǝtḥata śǝrāy is more commonly used than the translation and it might make sense to add its Geez title in parantheses